From 3bf95be744a66139f65bd3f97036fcd192694260 Mon Sep 17 00:00:00 2001
From: Bo Yang <bo.yang@amlogic.com>
Date: Tue, 16 Jun 2015 17:22:43 +0800
Subject: [PATCH 5659/5965] PD#107795: dts: g9tv,g9bb: disable watchdog for all
 boards

request from kieth.liu
agreed by kieth.liu, sandy.luo
approved by robin.zhu

Change-Id: Id62c5fd62bccc0272b683c7a1c420202a912a2b7
---
 arch/arm/boot/dts/amlogic/mesong9bb_n302.dtd     | 2 +-
 arch/arm/boot/dts/amlogic/mesong9bb_n303.dtd     | 2 +-
 arch/arm/boot/dts/amlogic/mesong9bb_n304.dtd     | 2 +-
 arch/arm/boot/dts/amlogic/mesong9bb_t826_pxp.dtd | 2 +-
 arch/arm/boot/dts/amlogic/mesong9bb_t826_skt.dtd | 2 +-
 arch/arm/boot/dts/amlogic/mesong9bb_t828_skt.dtd | 2 +-
 arch/arm/boot/dts/amlogic/mesong9tv_n300.dtd     | 2 +-
 arch/arm/boot/dts/amlogic/mesong9tv_n300_15g.dtd | 2 +-
 arch/arm/boot/dts/amlogic/mesong9tv_n300_1g.dtd  | 2 +-
 arch/arm/boot/dts/amlogic/mesong9tv_n301.dtd     | 2 +-
 arch/arm/boot/dts/amlogic/mesong9tv_pxp.dtd      | 2 +-
 arch/arm/boot/dts/amlogic/mesong9tv_skt.dtd      | 2 +-
 12 files changed, 12 insertions(+), 12 deletions(-)

diff --git a/arch/arm/boot/dts/amlogic/mesong9bb_n302.dtd b/arch/arm/boot/dts/amlogic/mesong9bb_n302.dtd
index 12ffbfd82628..adb1c429bf24 100644
--- a/arch/arm/boot/dts/amlogic/mesong9bb_n302.dtd
+++ b/arch/arm/boot/dts/amlogic/mesong9bb_n302.dtd
@@ -410,7 +410,7 @@
 ///	-	WATCHDOG
 	amlogic-watchdog {
 		compatible = "amlogic,aml-wdt";
-		status = "okay";
+		status = "disable";
 		default_timeout=<8>;
 		reset_watchdog_method=<1>;//0:sysfs,1:kernel
 		reset_watchdog_time=<2>;
diff --git a/arch/arm/boot/dts/amlogic/mesong9bb_n303.dtd b/arch/arm/boot/dts/amlogic/mesong9bb_n303.dtd
index 99d97fa41bc3..2029a6e01578 100644
--- a/arch/arm/boot/dts/amlogic/mesong9bb_n303.dtd
+++ b/arch/arm/boot/dts/amlogic/mesong9bb_n303.dtd
@@ -409,7 +409,7 @@
 ///	-	WATCHDOG
 	amlogic-watchdog {
 		compatible = "amlogic,aml-wdt";
-		status = "okay";
+		status = "disable";
 		default_timeout=<8>;
 		reset_watchdog_method=<1>;//0:sysfs,1:kernel
 		reset_watchdog_time=<2>;
diff --git a/arch/arm/boot/dts/amlogic/mesong9bb_n304.dtd b/arch/arm/boot/dts/amlogic/mesong9bb_n304.dtd
index 407f134771f1..2397de85ed98 100644
--- a/arch/arm/boot/dts/amlogic/mesong9bb_n304.dtd
+++ b/arch/arm/boot/dts/amlogic/mesong9bb_n304.dtd
@@ -410,7 +410,7 @@
 ///	-	WATCHDOG
 	amlogic-watchdog {
 		compatible = "amlogic,aml-wdt";
-		status = "okay";
+		status = "disable";
 		default_timeout=<8>;
 		reset_watchdog_method=<1>;//0:sysfs,1:kernel
 		reset_watchdog_time=<2>;
diff --git a/arch/arm/boot/dts/amlogic/mesong9bb_t826_pxp.dtd b/arch/arm/boot/dts/amlogic/mesong9bb_t826_pxp.dtd
index bf9ac75a8694..115576641b81 100644
--- a/arch/arm/boot/dts/amlogic/mesong9bb_t826_pxp.dtd
+++ b/arch/arm/boot/dts/amlogic/mesong9bb_t826_pxp.dtd
@@ -407,7 +407,7 @@
 ///	-	WATCHDOG
 	amlogic-watchdog {
 		compatible = "amlogic,aml-wdt";
-		status = "okay";
+		status = "disable";
 		default_timeout=<8>;
 		reset_watchdog_method=<1>;//0:sysfs,1:kernel
 		reset_watchdog_time=<2>;
diff --git a/arch/arm/boot/dts/amlogic/mesong9bb_t826_skt.dtd b/arch/arm/boot/dts/amlogic/mesong9bb_t826_skt.dtd
index f123fa2991a0..1a95deeedded 100644
--- a/arch/arm/boot/dts/amlogic/mesong9bb_t826_skt.dtd
+++ b/arch/arm/boot/dts/amlogic/mesong9bb_t826_skt.dtd
@@ -420,7 +420,7 @@
 ///	-	WATCHDOG
 	amlogic-watchdog {
 		compatible = "amlogic,aml-wdt";
-		status = "okay";
+		status = "disable";
 		default_timeout=<8>;
 		reset_watchdog_method=<1>;//0:sysfs,1:kernel
 		reset_watchdog_time=<2>;
diff --git a/arch/arm/boot/dts/amlogic/mesong9bb_t828_skt.dtd b/arch/arm/boot/dts/amlogic/mesong9bb_t828_skt.dtd
index e93f0de1918c..7e662fa665fc 100644
--- a/arch/arm/boot/dts/amlogic/mesong9bb_t828_skt.dtd
+++ b/arch/arm/boot/dts/amlogic/mesong9bb_t828_skt.dtd
@@ -418,7 +418,7 @@
 ///	-	WATCHDOG
 	amlogic-watchdog {
 		compatible = "amlogic,aml-wdt";
-		status = "okay";
+		status = "disable";
 		default_timeout=<8>;
 		reset_watchdog_method=<1>;//0:sysfs,1:kernel
 		reset_watchdog_time=<2>;
diff --git a/arch/arm/boot/dts/amlogic/mesong9tv_n300.dtd b/arch/arm/boot/dts/amlogic/mesong9tv_n300.dtd
index d662c8176622..7dd97583026a 100644
--- a/arch/arm/boot/dts/amlogic/mesong9tv_n300.dtd
+++ b/arch/arm/boot/dts/amlogic/mesong9tv_n300.dtd
@@ -419,7 +419,7 @@
 ///	-	WATCHDOG
 	amlogic-watchdog {
 		compatible = "amlogic,aml-wdt";
-		status = "okay";
+		status = "disable";
 		default_timeout=<8>;
 		reset_watchdog_method=<1>;//0:sysfs,1:kernel
 		reset_watchdog_time=<2>;
diff --git a/arch/arm/boot/dts/amlogic/mesong9tv_n300_15g.dtd b/arch/arm/boot/dts/amlogic/mesong9tv_n300_15g.dtd
index 14899993e066..e8c5ed2d861c 100644
--- a/arch/arm/boot/dts/amlogic/mesong9tv_n300_15g.dtd
+++ b/arch/arm/boot/dts/amlogic/mesong9tv_n300_15g.dtd
@@ -407,7 +407,7 @@
 ///	-	WATCHDOG
 	amlogic-watchdog {
 		compatible = "amlogic,aml-wdt";
-		status = "okay";
+		status = "disable";
 		default_timeout=<8>;
 		reset_watchdog_method=<1>;//0:sysfs,1:kernel
 		reset_watchdog_time=<2>;
diff --git a/arch/arm/boot/dts/amlogic/mesong9tv_n300_1g.dtd b/arch/arm/boot/dts/amlogic/mesong9tv_n300_1g.dtd
index 7883a6db8006..d4e88c1e30b3 100644
--- a/arch/arm/boot/dts/amlogic/mesong9tv_n300_1g.dtd
+++ b/arch/arm/boot/dts/amlogic/mesong9tv_n300_1g.dtd
@@ -407,7 +407,7 @@
 ///	-	WATCHDOG
 	amlogic-watchdog {
 		compatible = "amlogic,aml-wdt";
-		status = "okay";
+		status = "disable";
 		default_timeout=<8>;
 		reset_watchdog_method=<1>;//0:sysfs,1:kernel
 		reset_watchdog_time=<2>;
diff --git a/arch/arm/boot/dts/amlogic/mesong9tv_n301.dtd b/arch/arm/boot/dts/amlogic/mesong9tv_n301.dtd
index c3b99ba76d59..dc8857d2b982 100644
--- a/arch/arm/boot/dts/amlogic/mesong9tv_n301.dtd
+++ b/arch/arm/boot/dts/amlogic/mesong9tv_n301.dtd
@@ -419,7 +419,7 @@
 ///	-	WATCHDOG
 	amlogic-watchdog {
 		compatible = "amlogic,aml-wdt";
-		status = "okay";
+		status = "disable";
 		default_timeout=<8>;
 		reset_watchdog_method=<1>;//0:sysfs,1:kernel
 		reset_watchdog_time=<2>;
diff --git a/arch/arm/boot/dts/amlogic/mesong9tv_pxp.dtd b/arch/arm/boot/dts/amlogic/mesong9tv_pxp.dtd
index 20efa37f085c..6c68e24539b0 100644
--- a/arch/arm/boot/dts/amlogic/mesong9tv_pxp.dtd
+++ b/arch/arm/boot/dts/amlogic/mesong9tv_pxp.dtd
@@ -214,7 +214,7 @@
 ///	-	WATCHDOG
 	amlogic-watchdog{
 		compatible = "amlogic,aml-wdt";
-		status = "okay";
+		status = "disable";
 		default_timeout=<8>;
 		reset_watchdog_method=<1>;//0:sysfs,1:kernel
 		reset_watchdog_time=<2>;
diff --git a/arch/arm/boot/dts/amlogic/mesong9tv_skt.dtd b/arch/arm/boot/dts/amlogic/mesong9tv_skt.dtd
index 2b21dcc57665..5b4dd6394a9f 100644
--- a/arch/arm/boot/dts/amlogic/mesong9tv_skt.dtd
+++ b/arch/arm/boot/dts/amlogic/mesong9tv_skt.dtd
@@ -305,7 +305,7 @@
 ///	-	WATCHDOG
 	amlogic-watchdog {
 		compatible = "amlogic,aml-wdt";
-		status = "okay";
+		status = "disable";
 		default_timeout=<8>;
 		reset_watchdog_method=<1>;//0:sysfs,1:kernel
 		reset_watchdog_time=<2>;
-- 
2.19.0

